app/dashboard/templates/shared/issue_details.html

```diff
@@ -31,6 +31,13 @@ <h5 class="font-subheader">{% trans "Details" %}</h5>
     <option value="approval" selected>{% trans "Approval Required - I will approve bounty hunters." %}</option>
   </select>
 </div>
+<div class="w-100 mt-2" id="auto_approve_workers_container">
+  <div class="form__checkbox">
+    <input name="admin_override_suspend_auto_approval" id="admin_override_suspend_auto_approval" type="checkbox" value="1" checked />
+    <label class="form__label" for="admin_override_suspend_auto_approval" style="display: flex;">{% trans "Worker will be
```

Comment:
Is `style="display: flex;"` required here?

Can you push the {% trans %} on its own line?
```
<label ..>
  {% trans ".." %}
</label>
```